README.md

```diff
@@ -2,45 +2,44 @@
 
 ## Portfolio Setup Project
 
-This project is a replicate of figma mobile template one
+This project is a replicate of the figma mobile template one
 
 ### Getting Started
 
 1. copy the Url from my github account
-2. clone it to your local machine using
+2. clone it to your local machine
 
-### Prerequisites
+#### Prerequisites
 
 1. Setup
 2. Install
-3. Usage
-4. Tun tests
-5. Deployment
+3. Run tests
+4. Deployment
 
-### Built With
+#### Built With
 
 HTML and CSS
 
-### technology used
+#### technology used
 
 1.Linters
 2.VScode
 3.html & css
 4.github
 
-### Live Demo
+#### Live Demo
 
 **Link:** <http://127.0.0.1:5500/index.html>
 
-### Author
+#### Author
```

Comment:
- [ ] I think it would be better if you put your name in here. To let users know who is the creator of this website. 😊